### ISSUE TITLE
What do you think of a repo for meeting notes?

### ISSUE THREAD

**Issue by maddydeason** [WomenWhoCodeBoston/leaders-minutes#1]
@bryanaknight @ShaynaCummings @RachelMinto 

I thought it might be handy to have the notes from past leaders meetings in one place so that people who missed one could easily catch up?

**Comment by RachelMinto** [comment]
Sounds good to me. I have notes from the last meeting that I can post this evening.

**Comment by bryanaknight** [comment]
👍 we could also just have a general repo and have separate folders, including a meeting_notes/ folder. But i'm fine either way!

**Comment by maddydeason** [comment]
very true! might be a good way to consolidate things?

**Comment by bryanaknight** [comment]
Yeah and makes things more easily searchable in one repo vs across multiple repos. I think events warrants it's own but maybe any other docs, notes, assets could just go in one general repo

**Comment by maddydeason** [comment]
sounds good! what should we call this new repo? general? (not the best at naming if you can tell, haha)

**Comment by bryanaknight** [comment]
Haha, honestly cannot think of anything better than general. Maybe let's go with that and if someone comes up with something better we can always change the name.

**Comment by maddydeason** [comment]
haha sounds good to me!
